app/routes/inbox/meetings/new.js

```diff
@@ -0,0 +1,26 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class InboxMeetingsNewRoute extends Route {
+  @service currentSession;
+  @service router;
+  @service store;
+
+  beforeModel() {
+    if (!this.currentSession.canWrite) {
+      this.router.replaceWith('inbox.meetings');
+    }
+  }
+
+  model() {
+    let now = new Date();
+    let meeting = this.store.createRecord('zitting', {
+      geplandeStart: now,
+      gestartOpTijdstip: now,
+    });
+
+    return {
+      meeting,
```

Comment:
Not conventional but I wanted to get the conversation started:

I was thinking that _always_ returning an object (or a promise resolving to an object) from the model hook is a good default even if you only return 1 thing. I think it has 2 benefits compared to the alternative:
1. It sidesteps the issue that `@model` isn't very descriptive. `@model.meeting` is way more readable, at the cost of extra typing. That also means you don't _have_ to create a controller to alias things (or use the `{{let}}` helper to do it).
2. Once you do need extra model props in the route, you can simply add them, without having to start mass renaming all your `model` usage to include properties.

Now there are other solutions as well, but I think this kind of makes sense as a guideline since it doesn't stray away from the Ember way of doing things?

Opinions?

Reply:
not sure if I'm a fan, mostly because it makes it slightly harder to provide to model to a route with linkto (though we don't do that often).

Reply:
You mean providing the actual model instead of the record id? Yea, I didn't think of that, but I don't think that's a drawback :smile:.

Anyway, I figure you prefer to return the record directly instead?

Reply:
I have a slight preference for having the record directly with an alias indeed.

Reply:
Updated.